demos/sample_azure_iot_pnp/sample_azure_iot_pnp_device.c

```diff
@@ -0,0 +1,570 @@
+/* Copyright (c) Microsoft Corporation. All rights reserved. */
```

Comment:
THis seems to be simulated device implementation ... can we convey that in this implemtation file name?